cypress/support.ts

```diff
@@ -47,6 +48,7 @@ Cypress.Commands.add('install', () => {
       cy.byLegacyTestID('horizontal-link-Storage System').click();
       cy.byLegacyTestID('item-filter').type(`${OCS_SS}`);
       cy.get('td[role="gridcell"]', {timeout: 5 * 60000}).contains("Available");
+      cy.exec(OCS_SC_STATE, { timeout: 5 * 60000 });
```

Comment:
```suggestion
      cy.exec(OCS_SC_STATE, { timeout: 25 * 60000 });
```
StorageClusters can be incredibly slow to reach the state. Should increase the timeout to atleast 25 mins